src/plugins/intel_cpu/src/nodes/executors/acl/acl_reduce.cpp

```diff
@@ -71,15 +88,20 @@ bool AclReduceExecutor::init(const ReduceAttrs& reduceAttrs,
         case Algorithm::ReduceMin:
         case Algorithm::ReduceSum:
         case Algorithm::ReduceProd: {
-            Status reductionOperationStatus = NEReductionOperation::validate(&srcTensorInfo, &dstTensorInfo, axisCast(reduceAttrs.axes[0], srcDims.size()),
+            int axis = axisCast(reduceAttrs.axes[0], srcDims.size());
+            if (srcDescs[0]->hasLayoutType(LayoutType::nspc)) {
+                axis = getAxeNchwLayout(axis);
+                if (axis == -1) return false;
+            }
```

Comment:
Why do we need to duplicate the logic here and for ReduceMean?
Can we do it once before the switch case instead?

Reply:
Fixed